### DISCUSSION TITLE
How do I make a script that can open a program?

### DISCUSSION THREAD

**Discussion by AmodeusR** [johnlindquist/kit#1128] [Q&A] [answered]
I want to write a script that opens a program for me, like the terminal or code editor. Anything really. I looked at the Guide and Docs but it didn't say anything about how to open a program installed in my computer.

**Comment by AmodeusR** [comment, marked as answer]
Ok, I just discovered I can use a function called **open** and the url for the webpage or executable of what I want to open. If there's a more "correct way" of doing it, I'll be glad to hear, but if not, I think I don't need help anymore. Thanks anyway 😅

**Reply by AmodeusR** [reply]
If anyone is interested, the path for the neat windows terminal is this:
`C:\Users\[YourUsername]\AppData\Local\Microsoft\WindowsApps`

**Reply by johnlindquist** [reply]
@AmodeusR `open` is the most common way. It's just an alias for this library: https://www.npmjs.com/package/open
